exporter/opentelemetry-exporter-otlp-proto-grpc/setup.cfg

```diff
@@ -58,5 +58,7 @@ where = src
 [options.entry_points]
 opentelemetry_traces_exporter =
     otlp_proto_grpc = opentelemetry.exporter.otlp.proto.grpc.trace_exporter:OTLPSpanExporter
+opentelemetry_metrics_exporter =
+    otlp_proto_grpc = opentelemetry.exporter.otlp.proto.grpc.metric_exporter:OTLPMetricExporter
```

Comment:
Looking at this makes me wonder if there's an issue to track the work to produce an OTLPMetricExporter for proto over http

Reply:
Yes, https://github.com/open-telemetry/opentelemetry-python/issues/2447. I have started working on it.

Reply:
Glad to see 4 months ago me opened that issue :D